commands/moderation/say.js

```diff
@@ -7,14 +7,18 @@ module.exports = {
     usage: "<input>",
     run: (client, message, args) => {
         message.delete();
-        message.channel.send(args.join(" "));
+        if(args[0])
```

Comment:
Wouldn't you want to check the length of the array incase of an out of bounds exception. That's how I would tackle this in java but idfk how js arrays work.

like `args.length() > 0`